### DISCUSSION TITLE
Mantine next and prism when uploaded to new version 7.0.1?

### DISCUSSION THREAD

**Discussion by myagizmaktav** [mantinedev/mantine#4940] [Q&A]
Hi Mantine next and prism packages when uploaded to new version 7.0.1? @rtivital 

Another issue did you see my other request?
https://github.com/mantinedev/mantine-flagpack/issues/5
Thank you.

**Comment by rtivital** [comment]
You can find 7.0 changelog here – https://mantine.dev/changelog/7-0-0/
`@mantine/prism` package was replaced by `@mantine/core-highlight`

flagpack package will be migrated to 7.x later.
